docs/user-guide/concepts/agents/structured-output.md

```diff
@@ -167,41 +227,98 @@ class Person(BaseModel):
+print(result.structured_output.skills)                  # ["systems admin"]
+```
+
+
+### Agent-Level Default
+
+Set a default structured output model that applies to all agent invocations:
+
+```python
+class PersonInfo(BaseModel):
+    name: str
+    age: int
+    occupation: str
+
+# Set default structured output model for all invocations
+agent = Agent(structured_output_model=PersonInfo)
+result = agent("John Smith is a 30 year-old software engineer")
+
+print(f"Name: {result.structured_output.name}")      # "John Smith"
+print(f"Age: {result.structured_output.age}")        # 30
+print(f"Job: {result.structured_output.occupation}") # "software engineer"
+```
+
+!!! note "Note"
+    Since this is on the agent init level, not the invocation level, the expectation is that the agent will attempt structured output for each invocation.
+
+
+### Overriding Agent Default
+
+Override the agent's default structured output model for specific invocations that require different schemas:
+
+```python
+class PersonInfo(BaseModel):
+    name: str
+    age: int
+    occupation: str
+
+class CompanyInfo(BaseModel):
+    name: str
+    industry: str
+    employees: int
+
+# Agent with default PersonInfo model
+agent = Agent(structured_output_model=PersonInfo)
+
+# Override with CompanyInfo for this specific call
+result = agent(
+    "TechCorp is a software company with 500 employees",
+    structured_output_model=CompanyInfo
 )
 
-print(result.name)                    # "Jane Doe"
-print(result.address.city)            # "New York"
-print(result.contacts[0].email)       # "jane@example.com"
-print(result.skills)                  # ["systems admin"]
+print(f"Company: {result.structured_output.name}")      # "TechCorp"
+print(f"Industry: {result.structured_output.industry}") # "software"
+print(f"Size: {result.structured_output.employees}")    # 500
 ```
 
+
+
+
+
 Refer to Pydantic documentation for details on:
 
 - [Models and schema definition](https://docs.pydantic.dev/latest/concepts/models/)
 - [Field types and constraints](https://docs.pydantic.dev/latest/concepts/fields/)
 - [Custom validators](https://docs.pydantic.dev/latest/concepts/validators/)
 
+
 ### Error Handling
 
+In the event there is an issue with parsing the structured output, Strands will throw a custom `StructuredOutputException` that can be caught and handled appropriately:
+
 ```python
 from pydantic import ValidationError
+from strands.types.exceptions import StructuredOutputException
 
 try:
-    result = agent.structured_output(MyModel, prompt)
-except ValidationError as e:
-    print(f"Validation failed: {e}")
-    # Handle appropriately - options include:
-    # 1. Retry with a more specific prompt
-    # 2. Fall back to a simpler model
-    # 3. Extract partial information from the error
+    result = agent(prompt, structured_output_model=MyModel)
+except StructuredOutputException as e:
+    print(f"Structured output failed: {e}")
 ```
 
-### Async
+### Async Support
```

Comment:
Let's merge this into one of the earlier examples showing support - repeating this just to show that `agent()` and `agent.invoke_async()` work the same is a bit much